openvdb_wolfram/OpenVDBLink/Source/OpenVDBGrid/AggregateData.h

```diff
@@ -0,0 +1,207 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+#include "../Utilities/Aggregate.h"
```

Comment:
Please consider including an include guard in this file. I see that the following files are missing include guards:
```
OpenVDBGrid/AggregateData.h
OpenVDBGrid/CSG.h
OpenVDBGrid/DistanceMeasure.h
OpenVDBGrid/Filter.h
OpenVDBGrid/FogVolume.h
OpenVDBGrid/IO.h
OpenVDBGrid/Image.h
OpenVDBGrid/LevelSetCreation.h
OpenVDBGrid/Measure.h
OpenVDBGrid/Mesh.h
OpenVDBGrid/Morphology.h
OpenVDBGrid/Render.h
OpenVDBGrid/Setters.h
OpenVDBGrid/Transform.h
```

Reply:
This should be fixed now.